src/app/blog/smaller-is-better/page.mdx

```diff
@@ -0,0 +1,66 @@
+import { BlogPostLayout } from '@/components/BlogPostLayout'
+import { MotionCanvas } from '@/components/MotionCanvas'
+
+export const post = {
+  draft: false,
+  author: 'b5',
+  date: '2024-09-23',
+  title: 'Smaller is better',
+  description: 'In the upcoming v0.26.0 release, We\'re doubling down on iroh\'s networking stack as “what iroh is”, describing everything else as a custom protocol',
+}
+
+export const metadata = {
+    title: post.title,
+    description: post.description,
+    openGraph: {
+      title: post.title,
+      description: post.description,
+      images: [{
+        url: `/api/og?title=Blog&subtitle=${post.title}`,
+        width: 1200,
+        height: 630,
+        alt: post.title,
+        type: 'image/png',
+      }],
+      type: 'article'
+    }
+}
+
+export default (props) => <BlogPostLayout article={post} {...props} />
+
+
+We're excited to share a refinement in the way define iroh that we think will set more folks up for success. In the upcoming v0.26.0 release, **We're doubling down on iroh's networking stack as “what iroh is”, and describing everything else as a custom protocol.**
+
+We want to focus on making Iroh composable with other technologies, specifically *technologies that we don't make*. This will let us be explicit, vocal supporters of projects like Automerge, Willow, UCANs, and others that are doing great work in the p2p space.
```

Comment:
```suggestion
We want to focus on making iroh composable with other technologies, specifically *technologies that we don't make*. This will let us be explicit, vocal supporters of projects like Automerge, Willow, UCANs, and others that are doing great work in the p2p space.
```